semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/messaging/OtelMessagingSemanticConventionUtilsTest.java

```diff
@@ -81,4 +82,86 @@ public void testIsRabbitMqBackend() {
     v = MessagingSemanticConventionUtils.isRabbitMqBackend(e);
     Assertions.assertFalse(v);
   }
+
+  @Test
+  public void testGetRabbitmqDestination() {
```

Comment:
Worth adding tests for negative cases as well, where the operation/destination does not exist